lib/handler.js

```diff
@@ -17,8 +17,14 @@ if (GITHUB_API_ENDPOINT) {
   config.pathPrefix = url.parse(GITHUB_API_ENDPOINT).path;
 }
 
-module.exports = function (token, url, data, status, description) {
-  debug('Setting status', token, url, data, status, description)
+module.exports = function (token, url, data, status, description, context) {
+  debug('Setting status', token, url, data, status, description, context)
+  if(description === undefined) {
+    description = 'Strider test in progress';
+  }
+  if(context === undefined) {
+    context = 'strider-cd';
```

Comment:
Hmm, well this blog post is a little bit old.
Travis for example sets the following context:
 continuous-integration/travis-ci/pr